src/Avalonia.Base/Threading/Dispatcher.cs

```diff
@@ -83,42 +83,42 @@ public void RunJobs()
             _jobRunner.HasJobsWithPriority(minimumPriority);
 
         /// <inheritdoc/>
-        public Task InvokeAsync(Action action, DispatcherPriority priority = DispatcherPriority.Normal)
+        public Task InvokeAsync(Action action, DispatcherPriority priority = default)
```

Comment:
Note that Normal is now background rather than the most prioritized thing. Needed to prevent "window got closed inside [NSView updateLayer] callback" crashes